sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-bike-share/auto-ml-forecasting-bike-share.ipynb

```diff
@@ -0,0 +1,1037 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import (\n",
+    "    Environment,\n",
+    "    BatchEndpoint,\n",
+    "    BatchDeployment,\n",
+    "    BatchRetrySettings,\n",
+    "    Model,\n",
+    ")\n",
+    "from azure.ai.ml.constants import BatchDeploymentOutputAction\n",
+    "\n",
+    "model_name = \"bike-share\"\n",
+    "batch_endpoint_name = \"bike-share\"\n",
+    "\n",
+    "model = Model(\n",
+    "    path=f\"azureml://jobs/{best_run.info.run_id}/outputs/artifacts/outputs/model.pkl\",\n",
+    "    name=model_name,\n",
+    "    description=\"Bike share model.\",\n",
+    ")\n",
+    "registered_model = ml_client.models.create_or_update(model)\n",
+    "\n",
+    "env = Environment(\n",
+    "    name=\"automl-tabular-env\",\n",
+    "    description=\"environment for automl inference\",\n",
+    "    image=\"mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04:20210727.v1\",\n",
+    "    conda_file=\"artifact_downloads/outputs/conda_env_v_1_0_0.yml\",\n",
+    ")\n",
+    "\n",
+    "endpoint = BatchEndpoint(\n",
+    "    name=batch_endpoint_name,\n",
+    "    description=\"this is a sample batch endpoint\",\n",
+    ")\n",
+    "ml_client.begin_create_or_update(endpoint).wait()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To create a batch deployment, we will use the rolling_script.py which will load the model and will call the forecast method each time we will envoke the endpoint. The test set is longer than the forecast horizon specified at train time, so the forecasting script uses a so-called rolling evaluation to generate predictions over the whole test set. A rolling evaluation iterates the forecaster over the test set, using the actuals in the test set to make lag features as needed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output_file = \"rolling_forecast.json\"\n",
+    "batch_deployment = BatchDeployment(\n",
+    "    name = \"non-mlflow-deployment\",\n",
+    "    description = \"this is a sample non-mlflow deployment\",\n",
+    "    endpoint_name = batch_endpoint_name,\n",
+    "    model = registered_model,\n",
+    "    code_path = \"./forecast\",\n",
+    "    scoring_script = \"rolling_script.py\",\n",
+    "    environment = env,\n",
+    "    environment_variables = {\n",
+    "        \"TARGET_COLUMN_NAME\": target_column_name,\n",
+    "    },\n",
+    "    compute = cluster_name,\n",
+    "    instance_count = 2,\n",
+    "    max_concurrency_per_instance = 2,\n",
+    "    mini_batch_size = 10,\n",
+    "    output_action = BatchDeploymentOutputAction.APPEND_ROW,\n",
+    "    output_file_name = output_file,\n",
+    "    retry_settings = BatchRetrySettings(max_retries=3, timeout=30),\n",
+    "    logging_level = logging.INFO,\n",
```

Comment:
I am sorry, it looks like the backend do not like thid logging constant. Let us revert it to \"info\" and remove logging from imports.

Reply:
Reverted it back